test/user.test.js

```diff
@@ -363,7 +385,7 @@ describe("Update forum user successfully with valid ID", function() {
                                 assert.equal(res.body.updatedForumUser.username, "OldBob");
                                 assert.equal(res.body.updatedForumUser.displayName, "Bob");
                                 assert.equal(res.body.updatedForumUser.email, "joe@bobby.com");
-                                assert.equal(res.body.updatedForumUser.hashedPassword, "JoeBob");
+                                assert.equal(res.body.updatedForumUser.hashedPassword, hashPassword("JoeBob"));
```

Comment:
The ```hashPassword``` function export was accidently removed in the merge conflict fix so it requires being replaced at the bottom of ```models/user.server.model.js``` to fix the bug in this test

Reply:
Alright, thanks for letting me know.